Common/Data/Custom/SmartInsider/SmartInsiderIntention.cs

```diff
@@ -171,10 +171,54 @@ public override void FromRawData(string line)
             TimeProcessedUtc = string.IsNullOrWhiteSpace(tsv[41]) ? (DateTime?)null : ParseDate(tsv[41]);
             AnnouncedIn = string.IsNullOrWhiteSpace(tsv[42]) ? null : tsv[42];
 
-            Execution = string.IsNullOrWhiteSpace(tsv[43]) ? (SmartInsiderExecution?)null : JsonConvert.DeserializeObject<SmartInsiderExecution>($"\"{tsv[43]}\"");
-            ExecutionEntity = string.IsNullOrWhiteSpace(tsv[44]) ? (SmartInsiderExecutionEntity?)null : JsonConvert.DeserializeObject<SmartInsiderExecutionEntity>($"\"{tsv[44]}\"");
-            ExecutionHolding = string.IsNullOrWhiteSpace(tsv[45]) ? (SmartInsiderExecutionHolding?)null : JsonConvert.DeserializeObject<SmartInsiderExecutionHolding>($"\"{tsv[45]}\"");
-            ExecutionHolding = ExecutionHolding == SmartInsiderExecutionHolding.Error ? SmartInsiderExecutionHolding.SatisfyStockVesting : ExecutionHolding;
+            Execution = null;
+            if (!string.IsNullOrWhiteSpace(tsv[43]))
+            {
+                try
+                {
+                    Execution = JsonConvert.DeserializeObject<SmartInsiderExecution>($"\"{tsv[43]}\"");
+                }
+                catch (JsonSerializationException)
+                {
+                    Log.Error($"SmartInsiderIntention.FromRawData(): New unexpected entry found for Execution: {tsv[43]}. Parsed as Error.");
+                    Execution = SmartInsiderExecution.Error;
```

Comment:
This would mean this data point is malformed? Wonder if we should throw instead and have the caller discard it versus emitting invalid data points, what are your thoughts too @Jay-Jay-D

Reply:
This is a good approach, I think. Cases like this one are unexpected and - very - sporadic, we shouldn’t make the converter explode.

Reply:
Right 💯, my point was not making the converter explode but just discard this data point directly

Reply:
I'm thinking that we should model backtesting exactly as we would live trading. 

In the event of a sporadic signal, we might want to let the user deal with the data point rather than discarding it outright and introducing some sort of survivorship bias.